packages/sku/scripts/build-storybook.js

```diff
@@ -1,23 +1,20 @@
 // First, ensure the build is running in production mode
 process.env.NODE_ENV = 'production';
 
-const path = require('path');
 const { rimraf } = require('rimraf');
 const { argv } = require('../config/args');
 const gracefulSpawn = require('../lib/gracefulSpawn');
 const { storybookTarget } = require('../context');
 const buildStorybookPath = require.resolve('@storybook/cli/bin/index');
-const configDir = path.resolve(__dirname, '../config/storybook/build');
 const { runVocabCompile } = require('../lib/runVocab');
-const { setUpStorybookPreviewFile } = require('../lib/storybook');
+const { setUpStorybookConfigDirectory } = require('../lib/storybook');
 
 (async () => {
   await runVocabCompile();
   await rimraf(storybookTarget);
-  await setUpStorybookPreviewFile(configDir);
+  await setUpStorybookConfigDirectory();
```

Comment:
Given this is setting up a controlled folder with files on disk, was there a reason to do this separately to the `configure` script?

Reply:
I chose to associate it with the `storybook` commands because not all teams actually use Storybook. So this avoids apps that don't use Storybook getting an extra folder + file, even if it is gitignored.